spec/lib/tmuxinator/window_spec.rb

```diff
@@ -208,6 +210,69 @@
     end
   end
 
+  describe "#synchronize" do
+    context "synchronization enabled" do
+      let(:synchronize) { true }
+
+      it "should have synchronization enabled" do
+        expect(window.synchronize).to be true
+      end
+    end
+
+    context "synchronization disabled" do
+      let(:synchronize) { false }
+
+      it "should have synchronization enabled" do
+        expect(window.synchronize).to be false
+      end
+    end
+
+    context "synchronization not specified" do
+      it "should have synchronization disabled" do
+        expect(window.synchronize).to be false
+      end
+    end
+  end
+
+  describe "#tmux_synchronize_panes" do
+    let(:project) { double(:project) }
+    let(:window) { Tmuxinator::Window.new(yaml, 0, project) }
```

Comment:
Ah....  I think what you want here is an RSpec [shared context](https://relishapp.com/rspec/rspec-core/v/3-4/docs/example-groups/shared-context) to make that setup more DRY.

```ruby
describe Tmuxinator::Window do
  shared_context "new window setup" do
    let(:project) { double(:project) }
    let(:window) { Tmuxinator::Window.new(yaml, 0, project) }
    # etc
  end

  describe "#tmux_synchronize_panes" do
    include_context "new window setup"

    it "blah blah" do
    end
  end

  describe "#tmux_new_window_command" do
    include_context "new window setup"

    it "foo bar bazz" do
    end
  end
end
```

Probably a good idea, yes.